histFactory_hh/generatePlots.py

```diff
@@ -78,26 +78,26 @@ def getBinningStrWithMax(nBins, start, end, max):
             "mjjBinning": "3, { 0, 75, 140, 13000 }",
             "bdtNbins": 25
         },
-        "25x25": {
-            "mjjBinning": getBinningStrWithMax(25, 0, 600, 13000), 
-            "bdtNbins": 25
-        },
-        "20x20": {
-            "mjjBinning": getBinningStrWithMax(20, 0, 600, 13000), 
-            "bdtNbins": 20
-        },
-        "10x10": {
-            "mjjBinning": getBinningStrWithMax(10, 0, 600, 13000), 
-            "bdtNbins": 10
-        },
-        "10x25": {
-            "mjjBinning": getBinningStrWithMax(10, 0, 600, 13000), 
-            "bdtNbins": 25
-        },
-        "5x25": {
-            "mjjBinning": getBinningStrWithMax(5, 0, 600, 13000), 
-            "bdtNbins": 25
-        },
+        #"25x25": {
```

Comment:
For those of us not using fancy vim plugins, it's really easier if the '#' are aligned ;)